cmd/crane/cmd/append.go

```diff
@@ -18,28 +18,45 @@ import (
 	"log"
 
 	"github.com/google/go-containerregistry/pkg/crane"
+	"github.com/google/go-containerregistry/pkg/logs"
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/empty"
 	"github.com/spf13/cobra"
 )
 
 func init() { Root.AddCommand(NewCmdAppend()) }
 
 // NewCmdAppend creates a new cobra.Command for the append subcommand.
 func NewCmdAppend() *cobra.Command {
-	var baseRef, newTag, newLayer, outFile string
+	var (
```

Comment:
supernit: I find the `var (` format to be wasteful of vertical space except where it's more consistent at the top-level.

```go
var baseRef, newTag, outFile string
var newLayers []string
```

(up to you)

Reply:
This is the kind of pedantry I can get behind!